### ISSUE TITLE
Can't import figo

### ISSUE THREAD

**Issue by kisett** [figo-connect/python-figo#42]
I'm using the code for python. I can't run it, but i have installed figo, but it stops at the firts line, importing FigoSession. I get this error on the terminal

`Traceback (most recent call last):
  File "figoprueba.py", line 1, in <module>
    import figo
  File "/usr/local/lib/python3.4/dist-packages/figo/__init__.py", line 1, in <module>
    from .figo import *
  File "/usr/local/lib/python3.4/dist-packages/figo/figo.py", line 17, in <module>
    from requests_toolbelt.adapters.fingerprint import FingerprintAdapter
ImportError: No module named 'requests_toolbelt'`

I tryed to update pip and update python. Help please

**Comment by berend** [comment]
this error happens there is a missing dependency, in this case requests_toolbelt. It is part of the [requirements.txt](https://github.com/figo-connect/python-figo/blob/master/requirements.txt#L3) .

You can install this via `pip install -r requirements.txt`.

**Comment by berend** [comment]
Wait a second, now I understand the problem. By `pip install figo` all dependencies should be installed automatically, which does not happen now. We are going to fix this

**Comment by denizs** [comment]
In case you need a quickfix for this, you could also do something like this:
```python
from pip.req import parse_requirements

# we can use parse_requirements to obtain a generator to iterate over over our reqs
parsed_deps = parse_requirements('path/to/reqs')

deps = [str(d.req) for d in parsed_deps]

setup(
   install_requires=deps
)
```

Cheers, D

**Comment by denizs** [comment]
As a quickfix, you could also add `requests_toolbelt` to the `install_requires` list in the `setup.py` file like so:

```python
setup(
    ...
    install_requires = [
        'request_toolbelt'
    ]
)
```

**Comment by fricklerhandwerk** [comment]
This should now be fixed with #43.

**Comment by berend** [comment]
fixed with version 1.6.3, available via pip.
